### ISSUE TITLE
Business Details View

### ISSUE THREAD

**Issue by click2tman** [LamTechConsult/slbizr#44]

**Comment by developmenticon** [comment]
completed

**Comment by click2tman** [comment]
Yes. Most of this is complete. I don't see the map. When I click on bookmark,  claim business or edit business I expect to see another view that will be completed later. When I click on map or get directions or address row, I expect to be redirected to the views that will help give me directions. These can be blank for now but they are part of milestone 2

**Comment by click2tman** [comment]
This is fine I think. The only problem is that the map does not always show. It takes time to appear and in some cases does not appear. Can we use Google Static map of location and cache it to ensure it always displays? Remove "Bookmark" button for now and leave Claim and Edit only. The distance from users current location to business does not stick. We need to capture the users current location and save it in a variable locally and continue to use it until the user explicitly change it or their is a need for us to ask for a new location and update the saved value.

**Comment by developmenticon** [comment]
will fixed it soon.

**Comment by developmenticon** [comment]
This has been fixed, and for all map related issue.
